engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -53,6 +66,20 @@ import net.ceedubs.ficus.Ficus._
   lazy val callCacheReadActor = context.actorOf(RoundRobinPool(25)
     .props(CallCacheReadActor.props(callCache)),
     "CallCacheReadActor")
+  
+  // Docker Actor
+  lazy val ioEc = context.system.dispatchers.lookup(Dispatcher.IoDispatcher)
+  lazy val gcrQueriesPer100Sec = config.getAs[Int]("docker.gcr-api-queries-per-100-seconds") getOrElse 50000
```

Comment:
Hmm apparently the docs say 10 / sec https://cloud.google.com/container-engine/pricing
Which is clearly not 50000 / 100 sec 
We can tune it down to 1000 / 100 sec ?